packages/zoe/src/objArrayConversion.js

```diff
@@ -11,7 +13,7 @@ export const tuple = (...args) => args;
 
 /**
  * @template T
- * @template U
+ * @template {string | number} U
```

Comment:
If your intent is to include JS property names, it should include symbols. If your intent is narrower, why does it include numbers?

Reply:
Enumerable keys are what I'm aiming for, and `number` is included because in Typescript a `number` is never automatically coerced to a string (though it is, of course, when actually looking up a number-named property).

Reply:
Symbol-named properties can be enumerable